tests/Domain.UnitTests/Common/Exceptions/DomainValidationExceptionTests.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections.Generic;
+using CleanCompanyName.DDDMicroservice.Application.CommonTests.Builders;
+using CleanCompanyName.DDDMicroservice.Domain.Common.Exceptions;
+using CleanCompanyName.DDDMicroservice.Domain.Common.Validators;
+using FluentAssertions;
+using Xunit;
+
+namespace CleanCompanyName.DDDMicroservice.Domain.UnitTests.Common.Exceptions;
+
+public class DomainValidationExceptionTests
+{
+    [Fact]
+    public void WHEN_no_errors_happens_THEN_default_message_is_given()
+    {
+        var expectedMessage = "Some problems has been found during validation process:";
+        var result = new DomainValidationException(new List<ValidationError>());
```

Comment:
`result` is misleading IMO, I'd rather use `sut` (subject under test) or `domainValidationException`

Reply:
domainValidationException accepted